docs/content/en/docs/Reference/image_matrix.md

```diff
@@ -6,29 +6,22 @@ date: 2022-11-13
 description: >
 ---
 
-Kairos builds several artifacts for user convenience which are based on openSUSE, Alpine Linux, and Ubuntu.
+Kairos offers several pre-built images for user convenience based on popular Linux distributions such as openSUSE, Alpine Linux, and Ubuntu. The Kairos core team does its best to test these images, but those that are based on systemd (e.g. openSUSE, Ubuntu) are more thoroughly tested due to their homogenous settings. Support for other non-systemd based flavors (e.g. Alpine) may be limited due to team bandwidth. However, as Kairos is an open source community-driven project, we welcome any contributions, bug reports, and bug fixes. Check out our [Contribution guidelines](https://github.com/kairos-io/kairos/contribute) for more information.
```

Comment:
Debian is missing, just merged.

Reply:
Let's wait before adding it to the docs - we need still a release of it, better also wait from feedback after a release to be able to catch-up any issues that we couldn't spot in the CI before tagging